card/mod/01_core/set/all/content.rb

```diff
@@ -35,6 +34,10 @@ def clean_html?
   true
 end
 
+def history?
+  false
+end
```

Comment:
It means there is no history without the history mod. I override this with `true` in the history mod for all cards.